src/components/CompareResults/ResultsView.tsx

```diff
@@ -19,22 +19,15 @@ interface ResultsViewProps {
 }
 function ResultsView(props: ResultsViewProps) {
   const dispatch = useAppDispatch();
-  const {
-    baseRevInfo,
-    baseRepo,
```

Comment:
We don't need this anymore but I think we'll need it later, when moving the repository dropdown value out of redux. That's why I still decided to keep it returned from the loader.